Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -1496,11 +1496,17 @@ def panorama_get_address(address_name: str) -> Dict:
         'xpath': f'{XPATH_OBJECTS}address/entry[@name=\'{address_name}\']',
         'key': API_KEY
     }
-    result = http_request(
-        URL,
-        'GET',
-        params=params,
-    )
+
+    try:
+        result = http_request(
+            URL,
+            'GET',
+            params=params,
+        )
+    except Exception as err:
+        if OBJ_NOT_FOUND_ERR in str(err):
```

Comment:
do we not want to raise an error in this case, so users know something went wrong? 🤔

Reply:
I would prefer to handle it specifically in the `pan-os-get-address` command rather than in the general `http_request` which all the commands use.

Reply:
also note that the error 

```
Object was not found, verify that the name is correct and that the instance was committed.
```

is coming **after** the http_request, so it might not help to do try except here on this string.